setup/release.sh

```diff
@@ -0,0 +1,7 @@
+#!/bin/bash
+
+python ./setup/setup_idaplugin.py sdist --dist-dir ./setup/idaplugin_dist --formats=zip upload -r pypitest
+python ./setup/setup_server.py sdist --dist-dir ./setup/server_dist --formats=zip upload -r pypitest
+
+# cleanup
+rm -r ./setup/idaplugin_dist ./setup/server_dist ./rematch_idaplugin.egg-info ./server.egg-info
```

Comment:
1. Those paths are intentional. because of how setuptools works, the script must be executed from the repo's root, and because setup became several files I wanted to keep all of it in a separate subdirectory.
2. The release script is only fully usable by me, as it requires pypi credentials to the user owning the rematch-server and rematch-idaplugin packages. Do you still think it's necessary to have a python script?
3. Yes. That's correct. I'll fix this.